frontend/packages/kubevirt-plugin/integration-tests/tests/models/detailView.ts

```diff
@@ -25,12 +25,15 @@ export class DetailView {
       await browser.get(`${appHost}/k8s/ns/${this.namespace}/${this.kind}/${this.name}`);
       await isLoaded();
     }
-    await clickHorizontalTab(tabName);
-    await isLoaded();
+    if ((await $('.co-m-horizontal-nav-item--active').getText()) !== tabName) {
```

Comment:
This can be wrapped by a `view` function, like the rest of the code is doing.

Reply:
Done.